ktor-hosts/ktor-hosts-common/src/org/jetbrains/ktor/host/DefaultTransform.kt

```diff
@@ -19,8 +19,7 @@ fun ApplicationSendPipeline.installDefaultTransformations() {
                 ByteArrayContent(value)
             }
             is HttpStatusContent -> {
-                val contentType = call.defaultTextContentType(null)
-                TextContent("<H1>${value.code}</H1><P>${value.message.escapeHTML()}</P>", contentType, value.code)
+                TextContent("<H1>${value.code}</H1><P>${value.message.escapeHTML()}</P>", ContentType.Text.Html, value.code)
```

Comment:
It would also be nice to add a regression test. It was broken during introducing processing of charsets/contenttypes with `defaultTextContentType` and wouldn't be broken if there were a test :)